### PR TITLE
Theme Toggle And Contributing File

### PR DESCRIPTION
**Pull request by Programing-School**

### CODE REVIEW

**Review by mudit023 — CHANGES_REQUESTED** [review]
Add a readme file with a screen shot of the theme-toggle

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+3. Write your code.
+
+<!-- add any more guidelines for formatting and linting if required -->
+
+4. Before committing changes, you should verify if the issue is fixed
+
+5. Commit your changes. Try to follow the [best practices](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53) while committing changes.
+
+```bash
+git commit -s
```
Comment (mudit023) [inline CONTRIBUTING.md:44]: git commit -m "commit message"

Comment (mudit023) [inline CONTRIBUTING.md:44]: git commit -m "commit message"

**Review by mudit023 — COMMENTED** [review]
Correction in contributing.md

File: CONTRIBUTING.md (hunk 1 above)
Comment (Programing-School) [reply CONTRIBUTING.md:44]: Ok

**Pull request merged**